stingray/pulse/search.py

```diff
@@ -84,19 +98,25 @@ def epoch_folding_search(times, frequencies, nbin=128, segment_size=5000,
         correct for the exposure (Use it if the period is comparable to the
         length of the good time intervals.)
```

Comment:
Please add `weights` and `gti` to the docstring.

Reply:
Done